### PR TITLE
Update various python module provides

### PR DESCRIPTION
**Pull request by justinvreeland**
Fixes:
Missing dependency for multi version python3 subpackages

Related:

### CODE REVIEW

File: py3-click.yaml [hunk 1]
```diff
@@ -41,7 +41,10 @@ subpackages:
     description: ${{vars.pypi-package}} installed for python${{range.key}}
     dependencies:
       provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
       runtime:
+        - python-${{range.key}}
```
Comment (smoser) [inline py3-click.yaml:47]: you dont have to do this.
melange adds the correct depend here.  (which is actually `python-${{range.key}}-base`)

**Review by smoser — CHANGES_REQUESTED** [review]

**Review by smoser — CHANGES_REQUESTED** [review]

File: py3-cloudpickle.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package:
   name: py3-cloudpickle
   version: 3.0.0
-  epoch: 1
+  epoch: 2
   description: Extended pickling support for Python objects
```
[the remark this answers is not in the document]
Comment (justinvreeland) [reply py3-cloudpickle.yaml:5]: Nope, just missed it.

**Review by smoser — DISMISSED** [review]
I'm just going to approve to land.  We can fix the python dep removal later.

**Comment by justinvreeland** [comment]
@smoser think I'm still missing a review on this.

**Review by smoser — APPROVED** [review]

**Review by xnox — APPROVED** [review]

**Pull request merged**